supportedBackends/jes/src/test/scala/cromwell/backend/impl/jes/JesConfigurationSpec.scala

```diff
@@ -103,10 +116,6 @@ class JesConfigurationSpec extends FlatSpec with Matchers with TableDrivenProper
     new JesConfiguration(BackendConfigurationDescriptor(backendConfig, globalConfig)).root shouldBe "gs://my-cromwell-workflows-bucket"
   }
 
-  it should "have the correct default zones" in {
```

Comment:
default zones have been moved out of JesConfiguration to JesRuntimeAttributes, so the default zones should be checked in a different spec.